x-pack/plugins/siem/public/components/toasters/index.test.tsx

```diff
@@ -59,7 +59,7 @@ describe('Toaster', () => {
           if (toasts.length === 0) {
             dispatch({ type: 'addToaster', toast: mockToast });
           }
-        });
+        }, []);
```

Comment:
@XavierM 
I've added it because of previous version of enzyme had a gap in calling hooks.
Without adding `call this effect only after the first render` the test fails, because of when click the btn and dispatch `deleteToaster` -> the `toasts` array becomes empty and `useEffect` will return in the game and will add new toast.